docs/examples/index.md

```diff
@@ -0,0 +1,38 @@
+---
+id: index
+title: Overview
+---
+We write some example codes of `neon-js` for developers to refer to. The examples are expected to cover all cases and methods on different levels.
+
+## Cases
+- Network Initiation
```

Comment:
Please use this page to explain the structure rather than listing the cases. That is what the list of contents at the sidebar is for. Explain the high level groupings and what exactly are the constraints between high, low and very raw level APIs.

Reply:
@snowypowers Modified, please check. Any suggestions would be appreciated.